rest-api/src/main/java/gov/cms/qpp/conversion/api/model/Metadata.java

```diff
@@ -27,12 +27,18 @@
 	private Boolean overallStatus;
 	private Boolean conversionStatus;
 	private Boolean validationStatus;
-	private Boolean cpc;
+	private String cpc;
 	private String conversionErrorLocator;
 	private String validationErrorLocator;
 	private String rawValidationErrorLocator;
+
+	private Date createdDate;
 	private Boolean cpcProcessed;
 
+	public Metadata() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Document this public constructor by adding an explicit description. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUndocumentedApi)